### DISCUSSION TITLE
How to send alert to different destinations within the same project?

### DISCUSSION THREAD

**Discussion by kos31de** [getsentry/support#104] [Support] [answered]
Hello Folks,

Is it possible to send alerts to different destinations within the same project, depending on the type of error?
For example, for minor errors, only Slack notifications are needed, but for major errors, we would like to notify Opsgenie so that the phone rings.

Thank you.

**Comment by Kobby-Bawuah** [comment, marked as answer]
Hi @kos31de, take a look at our metrics alerts seen [here](https://docs.sentry.io/product/alerts/alert-types/#metric-alerts) to verify if it fits the use case you are describing.

**Reply by kos31de** [reply]
Thank you for your answer.
I wanted to separate the notifiers according to the type of error, so it looks like I can't use metrics.
However, I am no longer working on this issue, so it is ok.
